Mathlib/Tactic/ToAdditive/Frontend.lean

```diff
@@ -486,13 +485,15 @@ open Lean.Expr.FindImpl in
   and we're not remembering the cache between these calls. -/
 unsafe def additiveTestUnsafe (findTranslation? : Name → Option Name)
   (ignore : Name → Option (List ℕ)) (e : Expr) : Option Name :=
-  let rec visit (e : Expr) (inApp := false) : OptionT FindM Name := do
+  let rec visit (e : Expr) (inApp := false) : OptionT (StateM (PtrSet Expr)) Name := do
```

Comment:
I'm just have an extremely local understanding of the code, so take this with a grain of salt: this was `StateT` in Lean core; was this change intentional?

Reply:
Thanks for asking. It was `StateT (PtrSet Expr) Id`, which is the same as `StateM (PtrSet Expr)`

Reply:
:bulb: Makes sense, thanks!